code/game/objects/items/weapons/storage/backpack.dm

```diff
@@ -97,6 +97,74 @@
 	name = "trophy rack"
 	desc = "It's useful for both carrying extra gear and proudly declaring your insanity."
 	icon_state = "cultpack"
+	var/obj/item/organ/external/head/LS
+	var/obj/item/organ/external/head/MS
+	var/obj/item/organ/external/head/RS
+	var/mutable_appearance/LSma
+	var/mutable_appearance/MSma
+	var/mutable_appearance/RSma
+	var/placement
+
+/obj/item/storage/backpack/cultpack/attackby(obj/item/I, mob/living/user)
+	..()
+	if(istype(I, /obj/item/organ/external/head))
+		placement = input(user, "Where would you like to put the head?") in list(!LS ? "Left Spike" : "", !MS ? "Middle Spike" : "", !RS ? "Right Spike" : "", "Bag")
+		switch(placement)
+			if("Bag")
+				return
+			if("Left Spike")
+				if(!LS)
+					LS = I
+					LSma = mutable_appearance(I.icon, I.icon_state)
+					var/matrix/M = matrix()
+					M.Turn(335)
+					M.Translate(-4, 0)
+					I.transform = M
+					LSma.transform = M
+					add_overlay(LSma)
+				else return
```

Comment:
Instructions after an `if` or `else` are usually put on the next line. Even if it's just a one line instruction.
Best to keep to this standard for readability.
Also goes for the ones below

Reply:
Fixed